spec/models/preference_spec.rb

```diff
@@ -14,22 +14,15 @@
 #
 #  index_preferences_on_user_id  (user_id)
 #
-require 'rails_helper'
 
-RSpec.describe Preference, type: :model do
-  subject {
-    Preference.new(
-      name: "test",
-      description: "test",
-      restriction: false
-    )
-  }
-  it 'is valid with a name' do
-    expect(subject).to be_valid
-  end
+describe Preference do
+  describe 'validations' do
+    subject { build(:preference) }
+
+    let(:user) { create(:user) }
 
-  it 'is invalid without a name' do
-    subject.name = nil
-    expect(subject).to_not be_valid
+    it 'is valid with a name, description and restriction' do
```

Comment:
In model specs is nice to have tests that check a good build model is valid and also that validations are being triggered. 

In the basecode you have [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers) gem that may help you, something like `validate_presence_of(:name)` 😉